api/test_token_blacklist.sh

```
@@ -0,0 +1,146 @@
+# Set required environment variables
+export JWT_SECRET="test-jwt-secret-1234567890abcdefghijklmnopqrstuvwxyz-with-special-chars!@#$%"
+export AUTH0_DOMAIN="test-domain.auth0.com"
+export AUTH0_AUDIENCE="bocchi-the-map-api"
+export AUTH0_CLIENT_ID="test-client-id-12345"
+export ENV="development"
+export LOG_LEVEL="INFO" 
+export PORT="8081"
+
+echo "🔧 Building test server..."
+go build -o bin/blacklist-test-server ./cmd/test-server/main.go
+if [ $? -ne 0 ]; then
```

Shellcheck (0.10.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)